app/views/benefits/RemoveCompanyBenefitStopDateView.scala.html

```diff
@@ -14,58 +14,81 @@
  * limitations under the License.
  *@
 
-@import includes.{ptaHeader, cancelLink}
+@import includes.cancelLink
 @import uk.gov.hmrc.tai.forms.benefits.RemoveCompanyBenefitStopDateForm
-@import includes.forms.{inputRadioGroupStyling, submit}
 @import uk.gov.hmrc.tai.model.TaxYear
-@import uk.gov.hmrc.play.views.html.helpers._
+@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLegend
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
+@import includes.cancelLink
 
 @import uk.gov.hmrc.tai.util.{TaxYearRangeUtil => Dates}
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@this(main: MainTempleteNew, form: FormWithCSRF, govukRadios: GovukRadios, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
 
 @(stopDateForm: Form[Option[String]],benefitType:String, empName:String)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 @startOfCurrentTaxYear = @{Dates.formatDate(TaxYear().start)}
 @endOfCurrentTaxYear = @{Dates.formatDate(TaxYear().end)}
-@classArguments = @{if(stopDateForm(RemoveCompanyBenefitStopDateForm.StopDateChoice).value.getOrElse("") != RemoveCompanyBenefitStopDateForm.BeforeTaxYearEnd) "js-hidden data-stopDate inline form-field-group panel-indent" else "data-stopDate inline form-field-group panel-indent"}
 
-@header = @{
-    ptaHeader(
-        formForErrorSummary = Some(stopDateForm),
-        displayBackLink = true,
-        mainHeadingText = Messages("tai.benefits.ended.stopDate.heading", benefitType, empName),
-        preHeadingText = Messages("tai.benefits.ended.journey.preHeader"))
+@conditionalRadio = {
+    <div id="stopDate-container" class="govuk-body">
+        @Messages("tai.benefits.ended.stopDate.panel", startOfCurrentTaxYear, endOfCurrentTaxYear)
+    </div>
 }
 
+@main(
+    pageTitle = Some(Messages("tai.currentYearSummary.heading")),
+    title = Messages("tai.benefits.ended.stopDate.heading", benefitType, empName),
+    authedUser = Some(user),
+    backLinkContent = Some(Messages("tai.back-link.upper"))
```

Comment:
@masahmad I think we need to discuss the way we are doing backLinks. The issue we saw will happen here I presume as we are not passing in backLinkUrl. This will happen for all forms I believe

Reply:
To be fixed as part of a ticket